ob-pattern-1/templates/wso2ob-pattern-1-svcaccount.yaml

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: wso2svc-account
```

Comment:
@kdanW I think this name needs to be parameterized since, you have allowed the value to passed from `value.yaml` [here](https://github.com/wso2/kubernetes-open-banking/pull/13/files#diff-895dff208d6ba759c5f1ce6fdc113e84R93).

Reply:
Fixed with https://github.com/wso2/kubernetes-open-banking/pull/13/commits/a6c289bdb04eb294570ae486704103047497e89d